manifests/install.pp

```diff
@@ -53,13 +57,13 @@
       revision => $version,
     }
   } else {
-    package { 'letsencrypt':
+    package { $package_name:
```

Comment:
In the adding relationship case, wouldn't it be expected to have working `Package[$my_pkg]` when providing `$package_name`, eg. `class {'letsencrypt': package_name => $my_pkg}`? After all it is named "package_name".

I'm not really sure about the "right way" here ... *(reads puppet documentation)* ... I guess I don't find the aliasing feature as clean ;) So if your suggestion still stands, I'll add `name` parameter.